src/routes/docs/configuring/ftp/+page.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Connecting to FTP server
+---
+
+Files supports connecting to remote FTP and FTPs servers.
+This allows you to effortlessly connect to any server that supports these protocols, browse the directory structure, and perform tasks such as uploading, downloading, renaming, and deleting files or folders according to your needs.
```

Comment:
```suggestion
This allows users to effortlessly connect to any server that supports these protocols, browse the directory structure, and perform tasks such as uploading, downloading, renaming, and deleting files or folders according to your needs.
```
What does it mean by downloading and uploading?

Reply:
I was referring to downloading files from the server and uploading them to it. Although now I think we can skip that.

Reply:
Doesn't it automatically upload changes?

Reply:
Btw, it's best to leave conversations unresolved because otherwise it's hard to find the response.